airbyte-integrations/connectors/source-amazon-seller-partner/source_amazon_seller_partner/manifest.yaml

```diff
@@ -0,0 +1,1819 @@
+version: 6.5.0
+
+type: DeclarativeSource
+
+check:
+  type: CheckStream
+  stream_names:
+    - orders
+
+definitions:
+  authenticator:
+    type: CustomAuthenticator
+    class_name: source_amazon_seller_partner.components.AmazonSPOauthAuthenticator
+    client_id: "{{ config['lwa_app_id'] }}"
+    client_secret: "{{ config['lwa_client_secret'] }}"
+    refresh_token: "{{ config['refresh_token'] }}"
+    token_refresh_endpoint: "https://api.amazon.com/auth/o2/token"
+    host: "{{ config['endpoint'].replace('https://', '') }}"
+
+  base_requester:
+    type: HttpRequester
+    url_base: "{{ config['endpoint'] }}"
+    authenticator: "#/definitions/authenticator"
+    request_headers:
+      Amazon-Advertising-API-ClientId: '{{ config["client_id"] }}'
+
+  streams:
+    orders:
+      type: DeclarativeStream
+      name: Orders
+      primary_key:
+        - AmazonOrderId
+      ignore_stream_slicer_parameters_on_paginated_requests: true
+      incremental_sync:
+        type: DatetimeBasedCursor
+        cursor_field: LastUpdateDate
+        cursor_datetime_formats:
+          - "%Y-%m-%dT%H:%M:%SZ"
+        datetime_format: "%Y-%m-%dT%H:%M:%SZ"
+        start_datetime:
+          type: MinMaxDatetime
+          datetime: "{{ format_datetime(config['replication_start_date'], '%Y-%m-%dT%H:%M:%SZ') }}"
+          datetime_format: "%Y-%m-%dT%H:%M:%SZ"
+        end_datetime:
+          type: MinMaxDatetime
+          datetime: "{{ format_datetime(config.get('replication_end_date', now_utc() - duration('PT2M') ), '%Y-%m-%dT%H:%M:%SZ') }}"
```

Comment:
notes:
I'm not sure why we're subtracting 2 months from the current date when no `replication_end_date` is specified. I'll be removing this because I don't see it in the original Python implementation